watergun_assassin/authentication/views.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render
+from django.http import JsonResponse
+from django.contrib.auth.models import User
+from django.views.decorators.csrf import csrf_exempt
+
+
+@csrf_exempt
+def create_user(request):
+    try:
+        username = request.POST['username']
+        password = request.POST['password']
+        email = request.POST['email']
+        print(username,password,email)
```

Comment:
Let's remove this print statement, we shouldn't be leaving them in code that will make its way into long running branches. Eventually we'll get some logging put into place (though when we do we would never want to log out passwords).